challenges/challenge2.html

```diff
@@ -35,10 +35,13 @@ <h1 id='title' class='text-center'>Open-Source Solar System</h1>
       <div id='solar-system'>
         <!-- ADD Your Space-Objects Here -->
         <img id='davids_planet' src='../assets/img/planet.png' class='animated bounce infinite'>
+        <img id="emma_planet" src="../assets/img/jupiter.gif">
```

Comment:
Nice star & planet! The animation looks really good! 👌

Reply:
I don't know where else to ask this question, so I'm asking it here on a current pull request, but is there a place to ask general questions related to how to use git and GitHub?

I'm stuck.  You've merged my pull request for adding a camper cell.  I think that I need to:
(I've marked the tasks that I do not know how to do with an asterisk)

*Delete the branch I was using because it has been merged and I should open a new branch to work on Challenge 2?
*Sync my GitHub repository (that I forked from David's) with David's repository?
*Make my local folder on my own computer match David's repository? (first my GitHub, then his?)
*make a new branch from my VSCode?
work on Challenge 2
making commits locally on  my computer
pushing those commits to my repository on GitHub
making a pull request from my gitHub to David's GitHub (is this called upstream?)
wait for it to to be accepted and merged,
Then repeat.

I'd like the command for the asterisked items and actually all of them so I can check my knowledge.

Thank you so much, any who answer.

Reply:
Hi terrifricker!
Let's talk about this at https://gitter.im/Easy-To-Git/community (save this url somewhere)